src/sentry/runner/initializer.py

```diff
@@ -402,13 +402,19 @@ def validate_options(settings):
 def __model_unpickle_compat(model_id, attrs=None, factory=None):
     from django import VERSION
 
+    if attrs is not None or factory is not None:
```

Comment:
Basically this monkeypatch was to add the extra params in so that loading the old pickle wouldn't fail. It just wraps the new version of `model_unpickle` which doesn't accept those params. Assumption being that if we attempt to pass them then it's old models